cmd/lifecycle/exporter.go

```diff
@@ -259,6 +259,7 @@ func initDaemonImage(imagName string, runImageRef string, analyzedMD lifecycle.A
 		opts = append(opts, local.WithPreviousImage(analyzedMD.Image.Reference))
 	}
 
+	var appImage imgutil.Image
```

Comment:
Is this needed?

Reply:
Yes because `local.NewImage` now returns a `*local.Image` instead of an `imgutil.Image` type.  Without this, https://github.com/buildpacks/lifecycle/pull/485/files#diff-92c6ed289675df1e2edf598694dfc99f22a591496c130d10a43e86830cbd3686R282, fails because `cache.NewCachingImage` returns an `imgutil.Image` type and you can change the type of the variable without redeclaring it. However, since `*local.Image` implements `imgutil.Image` if you just declare the variable with the interface type everything works fine.

Reply:
This is very confusing but it looks like this is the way Golang works (because an interface is a pointer to itself).
Thanks for the clarification @ekcasey!